standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnDbUtil.java

```diff
@@ -180,6 +180,27 @@ public static void prepDb(Configuration conf) throws Exception {
         " WS_COMMIT_ID bigint NOT NULL," +
         " WS_OPERATION_TYPE char(1) NOT NULL)"
       );
+
+      stmt.execute("CREATE TABLE REPL_TXN_MAP (" +
+          " RTM_REPL_POLICY varchar(256) NOT NULL, " +
+          " RTM_SRC_TXN_ID bigint NOT NULL, " +
+          " RTM_TARGET_TXN_ID bigint NOT NULL, " +
+          " PRIMARY KEY (RTM_REPL_POLICY, RTM_SRC_TXN_ID))"
+      );
+
+      stmt.execute("CREATE TABLE \"APP\".\"SEQUENCE_TABLE\" (\"SEQUENCE_NAME\" VARCHAR(256) NOT " +
```

Comment:
Do we need these tables for ACID test? Also, we are not dropping them.

Reply:
yes its required as openTxn and all uses these tables. Added the drop for all.